webapp/src/main/java/com/box/l10n/mojito/service/thirdparty/ThirdPartyTMSSmartling.java

```diff
@@ -232,6 +328,10 @@ private TextUnitSearcherParameters baseParams(Long repositoryId,
         return result;
     }
 
+    long batchesFor(long total) {
```

Comment:
not clear at first sight what that is about. Rename? (or document?), test?

Reply:
Updated with tests.